src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -75,6 +75,9 @@ public static unsafe object RhNewArray(MethodTable* pEEType, int length)
         [RuntimeExport("RhBox")]
         public static unsafe object RhBox(MethodTable* pEEType, ref byte data)
         {
+            // A null can be passed for boxing of a null ref.
+            _ = Unsafe.ReadUnaligned<byte>(ref data);
```

Comment:
Add `ReadUnaligned<T>` to 
 src\coreclr\nativeaot\Runtime.Base\src\System\Runtime\CompilerServices\Unsafe.cs to fix this build break